specification/09-coveio_abi.adoc

```diff
@@ -320,7 +324,7 @@ first.
 
 `slot_id` must be between 0 and 7 inclusive.
 
-`cert_addr_out` must be page aligned.
+`cert_addr_out` must be 4KB page aligned.
```

Comment:
`4KB-aligned`, and ditto for the other changes below.

Reply:
yes, updated.